docs/culture/psychological-safety.md

```diff
@@ -0,0 +1,107 @@
+# Psychological Safety
+
+One of the primary reasons we host hackweeks is to help build a culture of collaboration. We do this because research is becoming increasingly interdependent, and because we know that cross-disciplinary exchanges increase the likelihood for us to address complex challenges. We also think that collaboration is fun and keeps our work more interesting and engaging!
+
+In order for collaboration to occur we must be able to share information, ask questions and express our opinions openly {cite:p}`edmondson_psychological_2023`. When we ask these questions of people outside of our disciplinary boundary, we might be stepping into situations were we feel incompetent or intrusive. This is where creating a psychologically safe environment can help.
+
+:::{admonition} Psychological Safety: Defined
+:class: note
+Psychological safety is a perception related to the consequences of taking *interpersonal risks* {cite:p}`edmondson_psychological_1999`. When people feel psychologically safe they can "express themselves physically, cognitively and emotionally" rather than disengage or "withdraw and defend their personal selves" {cite:p}`kahn_psychological_1990`.
+:::
+
+In this lesson we will work on developing more insight into risk and all the ways it might show up in a short format training setting. We'll do some exercises to assess our individual sense of risk, and build our capacity to assess how others might be experiencing risk in a classroom, small group and 1x1 learning setting. Then we will share strategies for increasing psychological safety by highlighting the positive outcomes that can occur when we willingly take risks.
```

Comment:
```suggestion
In this lesson we will work on developing more insight into risk and all the ways it might show up in a short format training setting. We'll do some exercises to assess our individual sense of risk and build our capacity to assess how others might be experiencing risk in a classroom, small group and 1x1 learning setting. Then we will share strategies for increasing psychological safety by highlighting the positive outcomes that can occur when we willingly take risks.
```

Reply:
@scottyhq @jomey Thoughts on the practice of putting each sentence on a newline? I seem to remember this made future diffs more manageable, particularly if we're not enforcing an 80/100 character max per line. @aaarendt I'd be happy to help make that change if we decide to go that route.

Reply:
> Thoughts on the practice of putting each sentence on a newline?

I have my editor set to wrap each line at 100 characters independent of a sentence. Might be worth adding a `.editorconfig` similar to the one we have for the [Jupyter Book template](https://github.com/uwhackweek/jupyterbook-template/blob/main/.editorconfig)